README.md

```diff
@@ -630,3 +631,24 @@ One of three out comes can occur after the export retried has completed.
 
 > *NOTE: Changing Writable.Pipeline.ExecutionOrder will invalidate all currently stored data and result in it all being removed from the database on the next retry.* This is because the position of the export function can no longer be guaranteed and no way to ensure it is properly executed on the retry.
 
+### Secrets
+
+Application Services can retrieve their secrets from the underlying secret store using the GetSecrets() API in the SDK. When security is disabled, the secrets are retrieved from the Writable.InsecureSecrets section of the configuration file. An example of the configuration for insecure secrets is below.
+
+#### Insecure Secrets Configuration
+
+```toml
+ [Writable.InsecureSecrets]    
+      [Writable.InsecureSecrets.CloudSecrets]
+        Path = '/v1/secret/edgex/appservice/cloud'
```

Comment:
path here should just be `cloud`. The rest is the base path in the `[SecretStore]` config

Reply:
I think a slightly better example is:
```
[Writable.InsecureSecrets.AWS]
path = 'aws'
[Writable.InsecureSecrets.AWS.Secrets]
username = 'aws-user'
password = 'aws-pw'